pkgs/applications/audio/airwindows/default.nix

```
@@ -0,0 +1,62 @@
+{ stdenv, fetchFromGitHub, requireFile, unzip, cmake }:
+
+stdenv.mkDerivation rec {
+  name = "airwindows-${version}";
+  version = "2020-05-25";
+
+  src = fetchFromGitHub {
+    owner = "airwindows";
+    repo = "airwindows";
+    rev = "897c0830e603a950e838a3753f89334bdf56bb71";
+    sha256 = "1mh7c176fdl5y5ixkgzj2j2b7qifp3lgnxmbn6b7bvg3sv5wsxg3";
+  };
+
+  vst-sdk = stdenv.mkDerivation rec {
+    name = "vstsdk366_27_06_2016_build_61";
+    src = requireFile {
+      name = "${name}.zip";
+      url = "https://www.steinberg.net/sdk_downloads/vstsdk366_27_06_2016_build_61.zip";
+      sha256 = "05gsr13bpi2hhp34rvhllsvmn44rqvmjdpg9fsgfzgylfkz0kiki";
+    };
+    nativeBuildInputs = [ unzip ];
+    installPhase = "cp -r . $out";
+  };
+
+  airwindows-ports = stdenv.mkDerivation rec {
+    name = "airwindows-ports";
+    src = fetchFromGitHub {
+      owner = "ech2";
+      repo = "airwindows-ports";
+      rev = "0.4.0";
+      sha256 = "1ya4qbc63sb52nzisdapsydrnnpqnjsl5kgxibbr3dxf32474g89";
+    };
+    installPhase = "cp -r . $out";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  patchPhase = ''
+    cd plugins/LinuxVST
+    rm build/CMakeCache.txt
+    mkdir -p include/vstsdk
+    cp -r ${airwindows-ports}/include/vstsdk/CMakeLists.txt include/vstsdk/
+    cp -r ${vst-sdk}/pluginterfaces include/vstsdk/pluginterfaces
+    cp -r ${vst-sdk}/public.sdk/source/vst2.x/* include/vstsdk/
+    chmod -R 777 include/vstsdk/pluginterfaces
+  '';
+
+  installPhase = ''
+    for so_file in *.so; do
+      install -vDm 644 $so_file -t "$out/lib/lxvst"
+    done;
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Handsewn bespoke linuxvst plugins";
+    homepage = http://www.airwindows.com/airwindows-linux/;
+    # airwindows is mit, but the vst sdk is unfree
+    license = licenses.unfree;
```

See earlier with the unfree license.
```suggestion
    license = licenses.mit;
```

---

Are you sure?  https://github.com/NixOS/nixpkgs/pull/45430#issuecomment-415561605
If not, I guess we don't need the separate license declaration for vst-sdk either, right?

---

https://github.com/NixOS/nixpkgs/pull/45430#issuecomment-415680981

> The best solution would be to make the vst sdk a separate unfree package and make airwave and airwindows reference it as a dependency. That would correctly describe the licensing situation.

is essentially what I'm suggesting. Adding `unfree` to just the internally used sdk derivation and setting this to `mit` accurately describes the licensing afaict, and I believe it will still act like an `unfree` package overall due to depending on one.

Since your linked PR, https://github.com/NixOS/nixpkgs/commit/dfc27a84302db6a2d798b99a8e35dedf2690a82b did just that: the sdk was marked as `unfree` and the main derivation itself continues being `mit`.

---

OK, thanks.
All done.